fileonfeature.txt

```diff
@@ -1,3 +1,5 @@
 This is my file on feature branch
 
-Added some lines to edit this file
\ No newline at end of file
+Added some lines to edit this file
+
+New line from new branch
```

Comment:
This is not the way we add new lines to our txt files!
It should start with "added" for example